server/fleet/windows_mdm.go

```diff
@@ -45,12 +45,13 @@ type MDMWindowsConfigProfile struct {
 // Returns an error if these conditions are not met.
 func (m *MDMWindowsConfigProfile) ValidateUserProvided() error {
 	if mdm.GetRawProfilePlatform(m.SyncML) != "windows" {
-		return errors.New("Only <Replace> supported as a top level element. Make sure you don’t have other top level elements.")
+		// it doesn't start with <Replace>, assume it isn't valid XML.
+		return errors.New("The file should include valid XML.")
 	}
 
 	doc := etree.NewDocument()
 	if err := doc.ReadFromBytes(m.SyncML); err != nil {
-		return fmt.Errorf("Couldn’t upload. The file should include valid XML: %w", err)
```

Comment:
I changed the approach so that `Couldn't upload` is always added in the single file upload API, so it doesn't show up when applying the spec from fleetctl.

Reply:
thanks and thanks for fixing the conditional above